UnitTests/GitUI.Tests/GitUI.Tests.csproj

```diff
@@ -11,6 +11,7 @@
     <ProjectReference Include="..\..\GitCommands\GitCommands.csproj" />
     <ProjectReference Include="..\..\GitExtUtils\GitExtUtils.csproj" />
     <ProjectReference Include="..\..\GitUI\GitUI.csproj" />
+    <ProjectReference Include="..\..\IntegrationTests\UI.IntegrationTests\UI.IntegrationTests.csproj" />
```

Comment:
Should we rather move `UITest.cs` to `CommonTestUtils`?

Reply:
❗️We mustn't have cross test refs.
We must move `CommitInfo` tests to UI.IntegrationTests project